### ISSUE TITLE
Accessibility page structure: Content header should have H2 and H3 tags

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#2623]
### Share your bug report, feature request, or comment.

Currently only our header has defined page structure. Given that the content header is a header with defining content we should give it header tags of H2 and H3. This should be for structure only and should _not_ change the style of the content header in any way.

![image](https://user-images.githubusercontent.com/414923/214395040-eca35866-ef95-46e0-987e-25f49a987b8a.png)

See #1826 for more accessibility discussion.
